### PR TITLE
Remove import-able generator files

### PR DESCRIPTION
**Pull request by kevmoo**
Fixes https://github.com/dart-lang/stagehand/issues/513

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 3.0.0
+
+- Move generator files until `lib/src`. Importing them directly is no longer
```
Comment (kwalrath) [inline CHANGELOG.md]: until into

File: lib/src/cli_app.g.dart [hunk 2]
```diff
@@ -10,4 +10,4 @@ part of 'cli_app.dart';
 // Generator: VersionGenerator
 // **************************************************************************
 
-const appVersion = '2.0.2';
+const appVersion = '3.0.0-dev';
```
Comment (kwalrath) [inline lib/src/cli_app.g.dart]: Why 3.0?

File: CHANGELOG.md (hunk 1 above)
Comment (kevmoo) [reply CHANGELOG.md]: ptal

**Comment by kevmoo** [comment]
PTAL

**Pull request merged**